scripts/pre_commit_linter.py

```diff
@@ -1376,6 +1378,8 @@ def _get_expression(parsed_node, components_to_check):
     # static expression.
     expression_callee_type_is_not_member = (
         expression['callee']['type'] != 'MemberExpression')
```

Comment:
Same thing, inline and explain. The substance of the comment above this is pretty good and much of it can be reused.

Reply:
done